j2lint/settings.py

```diff
@@ -1,6 +1,6 @@
 """settings.py - Settings class for jinja2 linter.
 """
-
+# pylint: disable=too-few-public-methods
 
 class Settings:
```

Comment:
Looking at how this class is used throughout the repo I am wondering if it is really useful (we won't refactor it as part of this PR). As far as I can see it only is really used when printing the `LintingError` in the `__repr__` method which is already a bit of an abuse of the `__repr__` method: https://docs.python.org/3/reference/datamodel.html#object.__repr__